README.md

```diff
@@ -371,37 +371,38 @@ An enum with the following values:
+| viewer-signaling     | signaling-viewer             | Calculated as the time  between creation of the `KinesisVideoClient` and the signaling client going to `open` state                                                    | Time taken to establish a signaling connection on the viewer-side                                                                                                               |
+| viewer-signaling     | setup-media-player-viewer           | Calculated as the time taken for `getUserMedia` call                                                                                                                   | Time taken to setup a media player on the viewer-side by seeking permissions for mic / camera (if needed), fetch tracks from the same and add them to the peer connection peer. |
+| viewer-signaling         | signaling-viewer-describe-channel                 | Calculated as the time taken for the `describeSignalingChannel` call                                                                                                   | Time taken for the API call to describeSignalingChannel on the viewer                                                                                                           |
+| viewer-signaling | signaling-viewer-describe-media-storage-config       | Calculated as the time taken for the `describeMediaStorageConfiguration` call                                                                                          | Time taken for the API call to describeSignalingChannel on the viewer                                                                                                           |
+| viewer-signaling | signaling-viewer-get-signaling-channel-endpoint  | Calculated as the time taken for the `getSignalingChannelEndpoint`                                                                                                     | Time taken for the API call to getSignalingChannelEndpoint on the viewer                                                                                                        |
+| viewer-signaling | signaling-viewer-get-ice-server-config                  | Calculated as the time taken for the `getIceServerConfig`                                                                                                              | Time taken for the API call to getIceServerConfig on the viewer                                                                                                                 |
+| viewer-signaling  | sign-connect-as-viewer                | Calculated as the time to sign the websocket url to connect to Signaling via connectAsViewer                                                                           | Time taken to sign the websocket url                                                                                                                                            |
```

Comment:
This is the new part ^, not sure why git decided to make the diff show like this but that's what we have to work with.